### DISCUSSION TITLE
Error: Unknown arg `content` in data.content for type TweetCreateInput.

### DISCUSSION THREAD

**Discussion by ghost** [prisma/prisma#13733] [Q&A] [answered]
Models: 
```prisma
model User {
  id          String    @id @default(uuid())
  handle      String    @unique
  displayName String
  email       String    @unique
  password    String
  bio         String?
  imageUrl    String?
  bannerUrl   String?
  followers   User[]    @relation("UserFollows", references: [id])
  following   User[]    @relation("UserFollows", references: [id])
  likes       Like[]
  tweets      Tweet[]
  comments    Comment[]
  retweets    Retweet[]
  createdAt   DateTime  @default(now())

}

model Tweet {
  id        String    @id @default(uuid())
  content   String
  user      User      @relation(fields: [userId], references: [id])
  userId    String
  like      Like[]
  comment   Comment[]
  createdAt DateTime  @default(now())
}
```
Query:
```ts
const newTweet = await prisma.tweet.create({
    data: {
        content: "hi",
            user: {
            connect: {
                id: user.id,
            },
        },
     },
 });
```
Error: 
```
PrismaClientValidationError: 
Invalid `prisma.tweet.create()` invocation:

{
  data: {
    content: 'hi',
    ~~~~~~~
    user: {
      connect: {
        id: '7a4344f9-8b26-4ad4-b04f-b2b6e6354109'
      }
    }
  }
}

Unknown arg `content` in data.content for type TweetCreateInput. Did you mean `createdAt`? Available args:
type TweetCreateInput {
  id?: String
  createdAt?: DateTime
  user: UserCreateNestedOneWithoutTweetsInput
}
```

But arg `content` shouldn't be unknown? Prisma studio shows this:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/63456887/172900269-06aefc5f-d811-468b-8bbc-c9139057cf03.png">

**Comment by austincrim** [comment, marked as answer]
Hey @mohantee 👋,

It looks like you've got everything set up correctly. 

Have you run `npx prisma generate`?
Have you tried restarting your code editor?

**Reply by ghost** [reply]
Restarting the code editor fixed it for me. Thanks.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
